vendor/github.com/mattn/go-shellwords/.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
   - tip
 before_install:
-  - go get github.com/axw/gocov/gocov
```

Comment:
Ah, I did not notice that v1.0.0 differs by this line, sorry for my inattention. But in this case there is some bug in trash IMO since it takes master go-shellwords while should v1.0.0 accordingly to `trash.conf`